services/galley/schema/src/V25.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes       #-}
+
+module V25 (migration) where
+
+import Cassandra.Schema
+import Text.RawString.QQ
+
+migration :: Migration
+migration = Migration 25 "Add conversation_codes table" $
+    schema' [r|
+        CREATE TABLE conversation_codes (
+            key           ascii, -- opaque conversation ID
+            conversation  uuid,
+            value         ascii, -- secret value
+            PRIMARY KEY (key, conversation)
```

Comment:
Perhaps it would be useful to add a `scope` here too to allow for other types of codes, similarly to the `vcodes` in `brig`?
Also, if `(key, conversation)` is the `PRIMARY KEY` then we can create the same `key` for different conversations so we should use only `key` as the `PRIMARY KEY`. Potentially, if `scope` is used, then a `PRIMARY KEY` like `(key, scope)` would make sense too.